spec/factories.rb

```diff
@@ -100,7 +100,7 @@
   factory :subscriber_list do
     sequence(:title) { |n| "title #{n}" }
     sequence(:slug) { |n| "title-#{n}" }
-    tags { { topics: { any: ["motoring/road_rage"] } } }
```

Comment:
Policies are a really old type of tag. Policies were retired, and you can no longer tag content to a `policy` anymore from Whitehall. In fact digging into documentation from previous teams, I think all policy subscriber lists were meant to have been bulk-unsubscribed. Similar to what we're doing here! So it might be safer to swap out `topics` with `taxons` instead. As taxons are the least likely to be retired AFAIK!

Reply:
The only possibile tags than can be used are policies and browse_pages, if I understand things correctly. Please, check this line: https://github.com/alphagov/publishing-api/blob/b08fbe0a3f58d827f38403c714920b69826f608f/content_schemas/dist/formats/email_alert_signup/notification/schema.json#L782

I think you are thinking about links, here: https://github.com/alphagov/publishing-api/blob/b08fbe0a3f58d827f38403c714920b69826f608f/content_schemas/dist/formats/email_alert_signup/notification/schema.json#L182

But this part of the tests checks tags, not links.

Reply:
Ah yep, taxons was not the right suggestion...

This is a SubscriberList factory. SubscriberLists are valid with any of these tags right? https://github.com/alphagov/email-alert-api/blob/main/lib/valid_tags.rb so you can pick any of these.

Reply:
OK, I think I get it now. Specialist-publisher bypasses publishing-api so I can use tags from it. I'll modify the code to use some of those.